### PR TITLE
New class is added

### PR DESCRIPTION
**Pull request by Milky34**
Review and verify the code madam

### CODE REVIEW

**Comment by Milky34** [comment]
send me updates

File: AshaPractice_github/src/main/java/Newpackage/Practice.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package Newpackage;
+
+public class Practice {
+
+	public static void main(String[] args) {
+
```
Comment (Milky34) [inline AshaPractice_github/src/main/java/Newpackage/Practice.java]: Review and verify the code, then update any rectification is there

**Review by Milky34 — COMMENTED** [review]
update madam

File: AshaPractice_github/src/main/java/Newpackage/Practice.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package Newpackage;
+
+public class Practice {
```
Comment (Asha306) [inline AshaPractice_github/src/main/java/Newpackage/Practice.java]: saptiyaaa

**Comment by Asha306** [comment]
use absulute path
don't use index
saptiyaa

File: AshaPractice_github/src/main/java/Newpackage/Practice.java (hunk 2 above)
Comment (Milky34) [reply AshaPractice_github/src/main/java/Newpackage/Practice.java]: haa tinnanu

**Review by Milky34 — COMMENTED** [review]
Thinnanu

**Comment by Milky34** [comment]
Madam meeru naanu code accept chesavaa

**Pull request closed without merging**